extensions/apple-reminders/src/my-reminders.tsx

```diff
@@ -82,7 +100,7 @@ export default function Command() {
             <Action.Push
               title="Create Reminder"
               icon={Icon.Plus}
-              target={<CreateReminderForm listId={listId} mutate={mutate} />}
+              target={<CreateReminderForm draftValues={draftValues} listId={listId} mutate={mutate} />}
             />
```

Comment:
I think we can add `clearSearchBar` when onPush. This way, after creating a reminder, the search bar will clear the previous search text.
```
<Action.Push
  title="Create Reminder"
  icon={Icon.Plus}
  target={<CreateReminderForm draftValues={draftValues} listId={listId} mutate={mutate} />}
  onPush={() => clearSearchBar()}
/>
```

Reply:
Thanks! Fixed that